helix-term/src/ui/picker.rs

```diff
@@ -365,9 +365,6 @@ impl<T> Picker<T> {
                     .map(|(index, _option)| (index, 0)),
             );
         } else if pattern.starts_with(&self.previous_pattern) {
-            // TODO: remove when retain_mut is in stable rust
-            use retain_mut::RetainMut;
```

Comment:
This will raise the version requirement to 1.61 though. You want to allow unused import instead